src/components/containers/ChartContainer.js

```diff
@@ -42,14 +47,13 @@ export default class extends React.Component {
 
             // The API provides y values as numbers between 0 and 1, but we want
             // to display them as percentages.
-            population.data.map(dataPoint => {
-                dataPoint.y = dataPoint.y * 100;
-                return dataPoint;
+            population.data.forEach(dataPoint => {
+                resultData.push({x: dataPoint.x, y: dataPoint.y * 100});
             });
 
             formattedData.datasets.push({
                 label: population.name,
-                data: population.data,
+                data: this.props.showOutliers ? resultData : this._removeOutliers(resultData),
```

Comment:
It looks like `this._removeOutliers(resultData)` will be called every time the button is pressed. Is that right? We could save the result to a variable somewhere above this line so that we don't end up re-doing the work.

Reply:
Yea that path led me to have this be 2 days late. I had a thing where we'd set an 'isInitialized' flag on the component and only do the sorting + `* 100` once but kept having issues with the component redoing that math causing 100000% scales and such since this function gets called on `render()`. I had it rewritten to use `willReceiveProps` but that led to other issues...granted it seems like the right approach and just probably needs more time.

Reply:
Gotcha. I understand. I did a similar "initialization" thing in EV and it made everything way more complicated.

If we can find a simple way to do it down the road, that's fine. But not worth stressing about.